scripts/build.js

```diff
@@ -141,6 +145,12 @@ function build(previousFileSizes) {
 
   console.log("Creating an optimized production build...");
 
+  const args = parseArgs(process.argv.slice(2));
+  if (args.analyze) {
+    console.log("And analyze bundle...");
+    config.plugins.push(new BundleAnalyzerPlugin());
```

Comment:
Why push this plugin here and not in webpack.config.js?

Reply:
I guess so that we load the plugin only if we want to use it ?